src/novo/src/client.tsx

```diff
@@ -18,7 +18,19 @@ import("v2/Artsy/Router/client")
       ReactDOM.hydrate(<ClientApp />, document.getElementById("react-root"))
     })
   })
-  .then(() => { return import("desktop/apps/authentication/client/initModalManager") })
-  .then(({ initModalManager }) => { initModalManager() })
-  .then(() => { mediator.on("auth:logout", logoutEventHandler) })
-  .catch(error => { console.error(error) })
+  .then(() => {
+    return import("desktop/apps/authentication/client/initModalManager")
+  })
+  .then(({ initModalManager }) => {
+    initModalManager()
+  })
+  .then(() => {
+    mediator.on("auth:logout", logoutEventHandler)
+  })
+  .catch(error => {
+    console.error(error)
+  })
```

Comment:
@icirellik - your refusal to use prettier is leading to churn in our git diffs

Reply:
Prettier runs on every commit? This is a major reason I didn't, and haven't, disable hooks globally.

Reply:
You must have used `--no-verify` then, as our prettier settings are [configured here](https://github.com/artsy/force/blob/master/package.json#L410-L417), which doesn't match the formatting of this block code prior to this commit.

Reply:
Rather than assume malice I'll propose more likely situations, it's much more likely that it occurred during a rebase or while my `node_modules` folder was deleted, I run `git clean -fdx` religiously when performance testing.

Either way, it appears that we should run prettier in CI this would ensure the issues are caught, and based on the thread from the other day it sounds like a good idea. It sounded like many engineers skip hooks or they may be skipped due to the package manager.